app/src/host.js

```diff
@@ -1,5 +1,7 @@
 /* eslint-disable no-console */
 /* eslint no-console: ["error", { allow: ["log"] }] */
+const serverUrl = process.env.NODE_ENV === 'development' ? 'http://localhost:8443' : 'https://www.toonin.ml'
```

Comment:
`${window.location.origin}` would be better instead of putting the actual url.

Reply:
How would the line change?

Reply:
oh I meant to for the false eval of the statement instead of putting 'https://www.toonin.ml' we can have `${window.location.origin}`